scripts/generate_psa_key_attributes.py

```diff
@@ -242,7 +248,7 @@ def main() -> None:
     if args.trng_key:
         value  = f'TRNG:{int(math.ceil(args.size / 8))}'
     elif args.key:
-        key = args.key.strip("0x")
+        key = remove_prefix(args.key, "0x")
```

Comment:
Nice catch.

Since Python 3.9, there's a built-in function to remove at most one occurrence of `0x`:
```suggestion
        key = args.key.removeprefix("0x")
```

Reply:
I thought as an extra service that we can remove any number of "0x" from the beginning of the string, but I can just put that function in a loop as I already do in the remove_prefix function.

Reply:
The reason the code was there in the first place is that I thought the validation code would not accept "0x" in the string, so just added the original code without giving it much though. When I tested now I discovered that it will accept "0x". Further down you see that I add 0x back to the string, because nrfutil assume that the string starts with "0x". So I can remove the code only accept properly formatted strings, so that is a string that starts with "0x" and then fail on anything else, but then I will have to check that the string starts with "0x" because that is the only variant nrfutil accepts.